changelog/next/features/4247--configured-pipeline-labels.md

```diff
@@ -0,0 +1,2 @@
+Pipelines configured as code in the `tenzir.yaml` configuration file may now
+contain labels.
```

Comment:
Please add that to the example in the docs.

Reply:
Already done:
https://github.com/tenzir/tenzir/pull/4247/commits/599d9776abb3a592c8c114921d5332f15adf8812#diff-86b97e9b4d26542ee0333dfd0fe11fc37947daffcb0d87b5d0eecc8948ffa6e9R106-R110